modules/ROOT/edits/datagraph-landing-page.adoc

```diff
@@ -0,0 +1,20 @@
+= Anypoint DataGraph Administration
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You manage Anypoint DataGraph in API Manager. The Anypoint DataGraph Administration tab for a particular environment appears only after you have added at least one API schema to that environment's unified schema.
```

Comment:
If this tab name is a label, boldface that label.
I'm sure you know this, so I will stop asking.  :-)

Reply:
It's actually a page, not a tab. And the it's expressed as DataGraph Administration in the UI. See commit below